agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java

```diff
@@ -83,12 +83,8 @@ public Span onPeerConnection(final Span span, final InetSocketAddress remoteConn
         onPeerConnection(span, remoteAddress);
       } else {
         // Failed DNS lookup, the host string is the name.
-        String hostString = remoteConnection.getHostString();
-        span.setAttribute(SemanticAttributes.NET_PEER_NAME.key(), hostString);
-        String peerService = mapToPeer(hostString);
-        if (peerService != null) {
-          span.setAttribute("peer.service", peerService);
-        }
+        String peerName = remoteConnection.getHostString();
+        setPeer(span, peerName, null);
```

Comment:
i think inlining this var is nice
```suggestion
        setPeer(span, remoteConnection.getHostString(), null);
```